CHANGELOG.develop

```diff
@@ -208,14 +210,16 @@ Benner and Paweł Siudak):
 - spacemacs-project
 - spacemacs-purpose (thanks to bmag)
 - spacemacs-visual
+**** Templates
+- Add templates layer (thanks to YasuharuIida and Eivind Fonn)
```

Comment:
Shall we move it to *Others* section, as Templates is a layer itself within *Completion layer* and wouldn't have any other sub layers?